src/app/views/about/about.view.html

```diff
@@ -0,0 +1,19 @@
+<ng-vertical-timeline [data]="events"></ng-vertical-timeline>
+<h1>My Hobbies & Interests</h1>
+<mat-grid-list cols="4" rowHeight="100px">
+  <mat-grid-tile
+      *ngFor="let tile of tiles"
+      [colspan]="tile.cols"
+      [rowspan]="tile.rows"
+      [style.background]="tile.color">
+  <div *ngIf="tile.text; else elseBlock">{{tile.text}}</div>
+  <ng-template #elseBlock><img src={{tile.image}} class= image></ng-template>
+  </mat-grid-tile>
+</mat-grid-list>
+<h1>You can download my resume here:
+  <a href="assets/downloads/resume01_27_21.pdf"
+  download="resume01_27_21.pdf">
+  resume01_27_21.pdf
+  </a>
+</h1>
```

Comment:
Having the resume at the bottom of the page makes it pretty inaccessible/hard to find. 
Recruiters might be looking for your resume. I think having this in the home page might be a better place.
```suggestion
  <a href="assets/downloads/resume01_27_21.pdf" download="resume01_27_21.pdf">
    Resume Download Link
  </a>
```